src/apps/aave/positions.ts

```diff
@@ -175,6 +175,7 @@ const hook: PositionsHook = {
             title: `Claimable rewards`,
             description: 'For supplying and borrowing',
             imageUrl: AAVE_LOGO,
+            manageUrl
```

Comment:
I don't see this for allbridge too. I assume this is for making positions clickable right? Should this be a required field in the types then? Seems like this should be present for all positions. Or if some positions don't have a url we can link to, can we enforce it in TS so that this field is explicitly specified as `undefined` rather than keeping it optional?

Reply:
@satish-ravi Addressed in 6718123ef43643c2d3a7daf9893a17db9cc5e699! Hopefully this more explicit definition will result in the `manageUrl` being provided more often.